mlflow/utils/conda.py

```diff
@@ -99,6 +99,11 @@ def get_or_create_conda_env(conda_env_path, env_id=None):
                    same conda dependencies but are supposed to be different based on the context.
                    For example, when serving the model we may install additional dependencies to the
                    environment after the environment has been activated.
+    :param capture_output: If True, captures conda env creation command stdout/stderr output and
+                           when conda command failed, attach captured stdout/stderr output to
+                           exception message and raise exception; if False, does not capture
+                           conda env creation command stdout/stderr output and when conda command
+                           failed, raise exception without stdout/stderr output attached.
```

Comment:
To me, this parameter description sounds a bit confusing because `get_or_create_conda_env` doesn't attach stdout/stderr to an exception but `process.exec_cmd` does.

Reply:
updated.